### ISSUE TITLE
Highlight doesn't work in "dark" theme in text cell

### ISSUE THREAD

**Issue by ghitsh** [HappenApps/Quiver#274]
Can you fix it asap? or let me know how to modify the built-in dark theme?

**Comment by ylian** [comment]
Yes I am aware. Not an easy fix. Highlight doesn't even add a class name, due to the limitation of html5 contenteditable. I have plan to revamp highlight in the future.
